raiden_contracts/deploy/contract_deployer.py

```diff
@@ -39,7 +39,7 @@ def __init__(
         web3: Web3,
         private_key: str,
         gas_limit: int,
-        gas_price: int = 1,
+        gas_price: int,
```

Comment:
It might make sense to adapt the Click parameter as well to only allow bigger than zero values:

```py
@click.option("--gas-price", default=5, type=int, help="Gas price to use in gwei")
```

Reply:
That's a good idea.